### PR TITLE
fix OpenJ9.gmk

### PR DESCRIPTION
**Pull request by gdams**
currently seeing the following build error:
`OpenJ9.gmk:203: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.`

### CODE REVIEW

File: closed/OpenJ9.gmk [hunk 1]
```diff
@@ -200,8 +200,8 @@ stage-j9 : \
 
 # Only update version files when the SHAs change.
 $(OUTPUT_ROOT)/vm/compiler/jit.version : $(call DependOnVariable, OPENJ9_SHA)
-        @$(MKDIR) -p $(@D)
-        $(ECHO) '#define TR_LEVEL_NAME "$(OPENJ9_SHA)"' > $@
+  @$(MKDIR) -p $(@D)
```
Comment (DanHeidinga) [inline closed/OpenJ9.gmk]: Should this be a tab?  The version for the old tr.source/jit.version file uses tab

Comment (jdekonin) [reply closed/OpenJ9.gmk]: Agreed, it appears to be 2 spaces which is still going to be a problem.

Comment (gdams) [reply closed/OpenJ9.gmk]: @DanHeidinga updated PTAL

**Comment by jdekonin** [comment]
Can you please add the 'signed off by' and more detailed comment.

**Comment by gdams** [comment]
@jdekonin updated PTAL

**Comment by DanHeidinga** [comment]
@gdams Thanks for fixing this so quickly!  Sorry I missed the tab vs space when merging the breaking change.

**Pull request merged**